pwa-devdocs/src/pwa-buildpack/troubleshooting/index.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Troubleshooting
+adobeio: /guides/packages/buildpack/troubleshooting/
```

Comment:
There's no troubleshooting page on the new site, but we should create on and simply point them to the [knowledgebase](https://support.magento.com/hc/en-us/search?query=pwa%20studio&hPP=20&idx=zendesk_production_magento_articles&p=0&utf8=%E2%9C%93&is_v=1) instead of copying this content over. I believe the knowledgebase team has already created entries for the items on this page, but I would double check with them.